packages/core/src/compile.ts

```diff
@@ -124,30 +120,24 @@ function serialiseObjectToType(v: any) {
   let result = '';
 
   for (const [key, value] of Object.entries(v)) {
-    if (value && typeof value === 'object') {
-      result += `'${encodeWithinSingleQuotes(key)}': ${serialiseObjectToType(
-        value,
-      )},`;
-    } else {
-      result += `'${encodeWithinSingleQuotes(key)}': ${value},`;
-    }
+    result += `${JSON.stringify(key)}: ${
```

Comment:
I think I understand this refactor, but it really confused me as it wasn't mentioned in the description and I wasn't sure why it was necessary.

Is this refactor relevant to the change?

Reply:
These changes were done along with the import/export of translation files we discussed. Since all the tests are passing I left the changes in.

Happy to move these changes to a separate PR.